repos/system_upgrade/el7toel8/actors/yumupgradability/actor.py

```diff
@@ -0,0 +1,43 @@
+from leapp.actors import Actor
+from leapp.models import Inhibitor
+from leapp.tags import IPUWorkflowTag, ExperimentalTag, InitRamStartPhaseTag
+from leapp.libraries.stdlib import call
+
+
+class YumUpgradability(Actor):
+    """
+    An actor that takes care of yum upgradability.
+    """
+
+    name = 'yum_upgradability'
+    consumes = ()
+    produces = (Inhibitor,)
+    tags = (IPUWorkflowTag, InitRamStartPhaseTag, ExperimentalTag)
+
+    def secure_yum_upgradability(self):
+        """
+        Move yum v3 data to yum v4 /etc/dnf directory
+
+        This is done in 3 steps
+            1) Create /etc/dnf directory
+            2) Copy yum data to the /etc/dnf directory
+            3) Remove all yum data from the /etc/yum directory
+
+        :return: bool
+        """
+        # Step 1
+        call(['mkdir', '/etc/dnf'])
+        # Step 2
+        call(['cp', '-a', '/etc/yum/*', '/etc/dnf/'])
+        # Step 3
+        call(['rm', '/etc/yum/*', '-rf'])
```

Comment:
This particular step will have additional consquences, but which are minimalized enough in case it will be moved into the  phase mentioned above, so I could live with that).